app/controllers/admin/analysis_controller.rb

```diff
@@ -9,14 +9,11 @@ class AnalysisController < AdminController
 
     before_action :check_aggregated_school_in_cache
     before_action :build_aggregate_school, except: [:analysis]
-    before_action :set_nav
     before_action :set_measurement_options
 
     def analysis
-      if @school.analysis?
-        # Redirect to correct dashboard
-        redirect_to admin_school_analysis_tab_path(school: @school, tab: pages.keys.first)
-      end
+      heat_meter = @aggregate_school.all_heat_meters.first
+      redirect_to admin_school_analysis_tab_path(@school, tab: :heating_model_fitting, mpan_mprn: heat_meter.mpan_mprn)
```

Comment:
this was redirecting to the overview page by default (now removed) so I've updated to redirect to the first meter found (and have subsequently removed all now unused nav and page code from the controller too)